examples/scala/src/main/scala/App.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.apache.spark.sql.{DataFrame, SparkSession}
+
+import com.microsoft.hyperspace._
+import com.microsoft.hyperspace.Hyperspace
+import com.microsoft.hyperspace.index.IndexConfig
+
+object App {
+  def main(args: Array[String]): Unit = {
+    // Create Spark session
+    val spark = SparkSession
+      .builder()
+      .appName("Hyperspace example")
+      .config("spark.some.config.option", "some-value")
```

Comment:
do we need this?

Reply:
Which line do you exactly mean? 
We need to create a SparkSession and instead of using local mode I switched to this way as Spark examples use this style. As an example, please look at [here](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/sql/SparkSQLExample.scala#L39).

Reply:
Usually, the line right above the comment: `config` in this case.

Reply:
Got it; yes I guess it is a good idea to keep this line to highlight the fact that user can change this to set master and create the Spark session in his desired mode, or add any other config override. But if you think it should be removed, I will drop it.